README.md

```diff
@@ -81,18 +81,22 @@ In order to run the local test setup, the following dependencies are needed:
 * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 
 ## Setup
+Install the latest minikube by following the instructions at:
+```https://minikube.sigs.k8s.io/docs/start/```
 
 To start minikube, issue the following command:
 
 ```shell-script
-minikube start --embed-certs --driver="virtualbox" --nodes 3
+minikube start --embed-certs --driver="virtualbox" --nodes 3 --network-plugin=cni --cni=calico 
```

Comment:
docker?

Reply:
Actually, should we specify a driver? Minikube should detect what's available.

Reply:
I could not get it to work with the virtualbox driver. There is some issue with the dns resolution not working in this configuration. Docker implementation seems ok